Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -304,6 +304,7 @@ - (void)setKeyboardType:(UIKeyboardType)keyboardType
 
 - (void)setShowSoftInputOnFocus:(BOOL)showSoftInputOnFocus
 {
+  (void)_showSoftInputOnFocus;
```

Comment:
Should we instead have the compiler avoid generating an ivar by adding an explicit getter? As it stands now if someone tries to access `showSoftInputOnFocus` they will always get `NO` regardless of what has been set.
```
- (BOOL)showSoftInputOnFocus
{
  return [[self backedTextInputView] inputView] == nil;
}
```
Alternatively, just declare a setter and not a property if it is not meant to be gotten

Reply:
I'm not familiar with the source code. I'm enabling the warning across the entire codebase for safeguard against future errors. Using `(void)decl;` seems like a safe approach. Could you suggest a patch i can use? I can also disable the warning in this build target as in the React/CoreModules below.